app/src/main/java/com/nutomic/syncthingandroid/activities/DeviceActivity.java

```diff
@@ -378,7 +378,7 @@ private void showDeleteDialog(){
     }
 
     private Dialog createDeleteDialog(){
-        return new android.app.AlertDialog.Builder(this)
+        return new AlertDialog.Builder(this, R.style.Theme_Syncthing_Dialog)
```

Comment:
According to my own tests, using values-night allows you to omit this (in multiple places)

Reply:
Is that better? If we make a values-night we have to pick the colors. But with the DayNight theme we can let Android pick the colors. I agree that it can be easy to accidentally forget to include this when creating dialogs in the future, but I like the idea of Android choosing how dark the dark theme is.

Reply:
Both solutions seem valid to me. If you'd like to keep the parameter, I'd suggest wrapping it into one maintainable place like https://github.com/Catfriend1/syncthing-android/commit/89b6bba293a30a360bdb6947d30e257a8862d1f5

Reply:
Okay it looks like we can compromise here. I don't think values-night is the magic here. It's just using the android:property in the xml. The fix I did for the settings menu pop ups seems to also apply to regular pop ups.

I think it still needs some tweaking though. Or we *should* still use the parameter. On my personal phone (API level 27) it works just fine. On my emulated API level 16 device the pop ups are dark themed but button text (again!!!) is the wrong color.

Reply:
I need to have a look at this on my devices to get a better impression... Thanks for your detailed testing so far :-)!

Reply:
I think the button text color is probably not the end of the world. It's using the accent color (yellow) instead of the primary color (blue). Lots of things in the app are already yellow. I think I'm fine with that the way it is. And I don't think there's much I can do about it. android:colorAccent is available starting with API level 21 and the minimum API level Syncthing supports is 16. So can't use that. I'll add the method to the utils thing and refactor all of the calls to use it. It'll be pretty tiny though. The DayNight theme eliminates the need to check theme before picking (although this won't be true once the black theme feature request comes in). So it'll only be 1 line.